licenses.html

```diff
@@ -249,14 +249,14 @@ <h1 class="text-white">FOSSASIA License Agreements</h1>
 						<div class="col-sm-7">
 							<span class="lead color-heading">About FOSSASIA</span>
 							<p>
-								FOSSASIA is the premier Free and Open Source technology organization in Asia for developers, designers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. The annual FOSSASIA took place in Cambodia, Vietnam and Singapore. FOSSASIA meetup groups exist throughout Asia.</p>
+								FOSSASIA develops Open Source software and hardware solutions with a global developer community from its base in Asia and organizes Open Technology events around the year. The organization also runs coding programs and development contests. The annual FOSSASIA Summit is the premier Open Technology event in Asia for developers, contributors, start-ups, and technology companies. FOSSASIA was founded in 2009 by <a target="_self" href="http://twitter.com/mariobehling" style="color:#777777">Mario Behling</a> and <a target="_self" href="http://twitter.com/hpdang" style="color:#777777">Hong Phuc Dang</a>.</p>
 						</div>
 
 						<div class="col-sm-3">
 							<ul class="contact-methods">
 								<li><i class="icon pe-7s-mail"></i><span><a href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
 								<li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-								<li><i class="icon pe-7s-map-marker"></i><span>Ms. DANG Hong Phuc</span><span>FOSSASIA Xoai Building, 93 Mau Than Street Xuan Khan, Ninh Kieu 9000 Can Tho, Vietnam, Asia</span></li>
+								<li><i class="icon pe-7s-map-marker"></i><span>Ms. DANG Hong Phuc</span><span>6 Eu Tong Sen Street #05-07 The Central, Singapore 059817</span></li>
```

Comment:
So basically there should be an `li` with _FOSSASIA_ before the updated address? @mariobehling

Reply:
Just see what works.